assets/src/frontend/components/printReceiptHtml.vue

```diff
@@ -122,6 +122,8 @@ export default {
     }
 
     .wepos-checkout-print-wrapper {
+        color: #181818;
+        font-family: monospace, monospace;
```

Comment:
You should always select a font first, if not found, then fallback to a default one. Use something like this:

```css
font-family: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
```